server/src/test/java/io/spine/server/event/given/EventRootCommandIdTestEnv.java

```diff
@@ -0,0 +1,375 @@
+
+        return ((AddTeamMember.Builder) Sample.builderForType(AddTeamMember.class))
+                .setTeamId(teamId)
+                .build();
+    }
+
+    public static AcceptInvitation acceptInvitation(TeamId teamId) {
+        checkNotNull(teamId);
+
+        final MemberInvitation invitation = memberInvitation(teamId);
+        return ((AcceptInvitation.Builder) Sample.builderForType(AcceptInvitation.class))
+                .setInvitation(invitation)
+                .build();
+    }
+
+    public static InviteTeamMembers inviteTeamMembers(TeamId teamId, int count) {
+        checkNotNull(teamId);
+
+        final InviteTeamMembers.Builder builder = InviteTeamMembers.newBuilder();
+        for (int i = 0; i < count; i++) {
+            final EmailAddress task = (EmailAddress) Sample.builderForType(EmailAddress.class)
+                                                           .build();
+            builder.addEmail(task);
+        }
+
+        return builder.setTeamId(teamId)
+                      .build();
+    }
+
+    private static MemberInvitation memberInvitation(TeamId teamId) {
+        return ((MemberInvitation.Builder) Sample.builderForType(MemberInvitation.class))
+                .setTeamId(teamId)
+                .build();
+    }
+
+    /**
+     * Creates a new {@link EventStreamQuery} without any filters.
+     */
+    public static EventStreamQuery allEventsQuery() {
+        return EventStreamQuery.newBuilder()
+                               .build();
+    }
+
+    public static class ProjectAggregateRepository
+            extends AggregateRepository<ProjectId, ProjectAggregate> { }
+
+    /**
+     * Routes the {@link ProjectCreated} event to the {@link TeamAggregate} the project belongs to.
+     * This is done for the purposes of the 
+     * {@link EventRootCommandIdShould#match_the_id_of_an_external_event_handled_by_an_aggregate()} 
+     * test.
+     */
+    @SuppressWarnings("SerializableInnerClassWithNonSerializableOuterClass")
+    public static class TeamAggregateRepository
+            extends AggregateRepository<TeamId, TeamAggregate> {
+
+        public TeamAggregateRepository() {
+            getEventRouting()
+                    .route(ProjectCreated.class,
+                           new EventRoute<TeamId, ProjectCreated>() {
+                               private static final long serialVersionUID = 0L;
+
+                               @Override
+                               public Set<TeamId> apply(ProjectCreated msg, EventContext ctx) {
+                                   return singleton(msg.getTeamId());
+                               }
+                           });
+        }
+    }
+
+    /**
+     * Routes the {@link InvitationAccepted} event to the {@link TeamCreationProcessManager} which 
+     * created the invitation. This is done for the purposes of the 
+     * {@link EventRootCommandIdShould#match_the_id_of_an_external_event_handled_by_a_process_manager()} 
```

Comment:
` {@link EventRootCommandIdShould#match_the_id_of_an_external_event_handled_by_a_process_manager()} 
 test.`
would better look like this:
```java
 {@linkplain EventRootCommandIdShould#match_the_id_of_an_external_event_handled_by_a_process_manager() test}.
```